tests/controllers/index.ts

```diff
@@ -1 +1,148 @@
-export * from './ControllerFactory';
+import { Connection } from 'typeorm';
+import FeedbackService from '../../services/FeedbackService';
+import { FeedbackController } from '../../api/controllers/FeedbackController';
+import { UserController } from '../../api/controllers/UserController';
+import UserAccountService from '../../services/UserAccountService';
+import StorageService from '../../services/StorageService';
+import { AdminController } from '../../api/controllers/AdminController';
+import AttendanceService from '../../services/AttendanceService';
+import { AttendanceController } from '../../api/controllers/AttendanceController';
+import { AuthController } from '../../api/controllers/AuthController';
+import { EventController } from '../../api/controllers/EventController';
+import { LeaderboardController } from '../../api/controllers/LeaderboardController';
+import { MerchStoreController } from '../../api/controllers/MerchStoreController';
+import UserAuthService from '../../services/UserAuthService';
+import EmailService from '../../services/EmailService';
+import EventService from '../../services/EventService';
+import MerchStoreService from '../../services/MerchStoreService';
+
+export class ControllerFactory {
```

Comment:
This class is getting messy. Let's remove the controller singletons and initialize a new controller each time the factory's called.

Reply:
True. Is the reason we are willing to trade-off readability for memory because this class is used only in tests and isn't persisting any memory afterwards? (meaning we are okay using a small amount more of memory since it's all present only during the tests running)